test/api_test.py

```diff
@@ -10,39 +11,30 @@
 
 
 class GlusterdConfig(object):
-    """
-    GlusterdConfig Class
-    """
+    """GlusterdConfig Class."""
```

Comment:
@devyanikota this doc string can be better written. Something like "Class to test the methods." or "Class to test the bindings" ...